x/authz/msgs.go

```diff
@@ -42,19 +42,20 @@ func NewMsgGrant(granter sdk.AccAddress, grantee sdk.AccAddress, a Authorization
 }
 
 // GetSigners implements Msg
-func (msg MsgGrant) GetSigners() []string {
-	return []string{msg.Granter}
+func (msg MsgGrant) GetSigners() []sdk.AccAddress {
+	granter, _ := sdk.AccAddressFromBech32(msg.Granter)
+	return []sdk.AccAddress{granter}
 }
 
 // ValidateBasic implements Msg
 func (msg MsgGrant) ValidateBasic() error {
 	granter, err := sdk.AccAddressFromBech32(msg.Granter)
 	if err != nil {
-		return sdkerrors.Wrap(sdkerrors.ErrInvalidAddress, "invalid granter address")
+		return sdkerrors.ErrInvalidAddress.Wrapf("invalid granter address: %s", err)
```

Comment:
Are we wrapping `err`? If so, shouldn't it be `%w`?

Reply:
I tried this initially, but when you run tests, it'll output something like this: `x/gov/types/msgs.go:149:10: Wrapf call has error-wrapping directive %w` for every line that has that 👎🏻 

Pretty sure this is because `%w` is for things like `fmt.Errorf(....)` whereas the SDK's `Wrapf` method is using `Sprintf` under the hood